.github/workflows/publish_code.yml

```diff
@@ -10,7 +11,6 @@ jobs:
   pypi:
     name: pypi
     runs-on: ubuntu-latest
-    needs: github
```

Comment:
This was a left-over dependency I forgot to remove. I used to have two jobs in here, the first one was called `github` and the `pypi` job would only run once the other had passed.